src/rosdep2/sources_list.py

```diff
@@ -439,7 +439,8 @@ def _generate_key_from_urls(urls):
 
 
 def update_sources_list(sources_list_dir=None, sources_cache_dir=None,
-                        success_handler=None, error_handler=None):
+                        success_handler=None, error_handler=None,
+                        include_eol_distros=False):
```

Comment:
The new parameter can be added to the docstring.

Reply:
Good point: f1ba42dc68f895dadcc18350d228eb62cff01854